src/chains/ethereum/transaction/src/eip1559-fee-market-transaction.ts

```diff
@@ -178,6 +172,9 @@ export class EIP1559FeeMarketTransaction extends RuntimeTransaction {
       }
     };
   }
+  public calculateIntrinsicGas(): bigint {
+    return super.calculateIntrinsicGas() + this.accessListDataFee;
```

Comment:
This is a bug fix. We weren't include these costs in intrinsic costs when we should.